packages/docs/src/lang/en/framework/Grid.json

```diff
@@ -87,6 +87,8 @@
       "alignSelfBaseline": "Apply `align-self: baseline`\n\n[MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/align-self)",
       "grow": "Apply `flex-grow: 1`\n\n[MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow)",
       "shrink": "Apply `flex-shrink: 1`\n\n[MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink)",
+      "grow-0": "Apply `flex-grow: 0`\n\n[MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-grow)",
+      "shrink=0": "Apply `flex-shrink: 0`\n\n[MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink)",
```

Comment:
`shrink-0`

Reply:
good catch; actually should be `grow-shrink-0`. Updated in https://github.com/vuetifyjs/vuetify/pull/6538